net/openfortivpn/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        adrienverge openfortivpn 1.6.0 v
+github.tarball_from tags
```

Comment:
Why use the tag when there are official release artifacts on github?

Reply:
@millerdev `github.tarball_from` defaults to `tags`, so you should remove this line.

@pmetzger Where are the official release artifacts? I can't find any on https://github.com/adrienverge/openfortivpn/releases.

Reply:
Things like https://github.com/adrienverge/openfortivpn/archive/v1.7.0.tar.gz don't count? (Maybe I'm mistaken.( In either case, the `github.tarball_from` line isn't needed.

Reply:
@pmetzger That is one of two URLs of tarballs generated by GitHub/Git, and the other is the default (`tags`) used by the github PortGroup. It doesn't count as release artifacts, for which we use `releases`.

https://github.com/macports/macports-ports/blob/1f5e8dc73a1c2c2b7b4f1bf6314b2d183b1472bc/_resources/port1.0/group/github-1.0.tcl#L32-L46